commons/connector/fitbit/fitbit_intraday_spo2.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitIntradaySpo2",
+  "doc": "Intra day spo2 data from fitbit device.",
```

Comment:
Can you please add more info from the webapi docs (esp regarding that spo2 is mainly calculated during sleep periods)